codegen/templates/endpoint_test.tmpl

```diff
@@ -32,7 +32,7 @@ func Test{{.HandlerID | Title}}{{.TestName | Title}}OKResponse(t *testing.T) {
 	gateway, err := testGateway.CreateGateway(t, nil, &testGateway.Options{
 		KnownHTTPBackends: []string{"{{$clientName | camel}}"},
 		TestBinary: filepath.Join(
-			getDirName(), "..", "..", "main.go",
+			getDirName(), "..", "..", "services", "example-gateway", "main.go",
```

Comment:
^ We need the service name available in the template here. Hard coding it to `example-gateway` will break `edge-gateway`.

For `edge-gateway` it should be `"edge-gateway"`.

We can figure out multiple binaries / gateways in a seperate PR.